neophrase.gemspec

```
@@ -1,50 +1,38 @@
-  # File lists for gem packaging
-  spec.files         = Dir.glob("lib/**/*") + %w[README.md LICENSE]
+  spec.files         = Dir["lib/**/*"] + %w(README.md LICENSE)
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  # Ruby compatibility
   spec.required_ruby_version = '>= 2.6'
 
-  # Runtime dependencies
   spec.add_runtime_dependency 'ruby-openai', '~> 3.7'
   spec.add_runtime_dependency 'loco_strings', '~> 0.1.4.1'
```

rubocop

[convention] Dependencies should be sorted in an alphabetical order within their section of the gemspec. Dependency `loco_strings` should appear before `ruby-openai`.

(Gemspec/OrderedDependencies)